templates/hdm.yml.epp

```diff
@@ -6,6 +6,7 @@
   <%- $hdm::puppetdb_settings.each |$key, $value| { -%>
     <%= $key %>: <%= $value %>
   <%- } -%>
+  hiera_config_file: "<%= $hdm::hdm_hiera_config_file %>"
```

Comment:
Looks to be no tests for the contents of this configuration file - add some?

Reply:
added a minimal set of spec tests